specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2257,6 +2257,11 @@
             "modelAsString": true
           },
           "description": "Set the minimum TLS version to be permitted on requests to storage. The default interpretation is TLS 1.0 for this property."
+        },
+        "isNfsV3Enabled": {
+          "type": "boolean",
+          "x-ms-client-name": "IsNfsV3Enabled",
```

Comment:
>IsNfsV3Enabled [](start = 31, length = 14)

How will customer able to see this parameter value?

Reply:
I am also curious about the x-ms-client-name. It seems no difference with property name. Why we need such an extension to define?

Reply:
I copied it from isHnsEnabled. Will rename it to EnableNfsV3